internal/testcafe/config.go

```diff
@@ -76,8 +77,18 @@ func FromFile(cfgPath string) (Project, error) {
 		return p, fmt.Errorf("failed to parse project config: %v", err)
 	}
 
-	if p.Testcafe.ProjectPath == "" {
-		return p, fmt.Errorf("no project folder defined")
+	if p.Testcafe.ProjectPath == "" && p.RootDir == "" {
+		return p, fmt.Errorf("could not find 'rootDir' in config yml, 'rootDir' must be set to specify project files")
+	}
+	if p.Testcafe.ProjectPath != "" {
```

Comment:
```suggestion
	if p.Testcafe.ProjectPath != "" && p.RootDir == "" {
```

Otherwise you will get twice the deprecated message.

Reply:
✅